android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/FxSuggestStorage.kt

```diff
@@ -25,7 +25,7 @@ class FxSuggestStorage(
     // Lazily initializes the store on first use. `Context.getDatabasePath()` does I/O, so
     // `store.value` should only be accessed from the read or write scope.
     private val store: Lazy<SuggestStore> = lazy {
-        SuggestStore(context.getDatabasePath(DATABASE_NAME).absolutePath)
+        SuggestStore(context.cacheDir.absolutePath)
```

Comment:
Almost! `getDatabasePath` returned a `File` for the path that includes the database name, but now I think we need to build that path ourselves, using something like:

```suggestion
        SuggestStore(File(context.cacheDir, DATABASE_NAME).absolutePath)
```

Reply:
Oops, thanks for the catching that. I was too quick. 😅